pylint/checkers/similar.py

```diff
@@ -117,11 +417,11 @@ def _display_sims(self, sims):
             print()
             print(num, "similar lines in", len(couples), "files")
             couples = sorted(couples)
-            lineset = idx = None
-            for lineset, idx in couples:
-                print(f"=={lineset.name}:{idx}")
+            lineset = start_line = end_line = None
+            for lineset, start_line, end_line in couples:
+                print(f"=={lineset.name}:[{start_line}:{end_line}]")
```

Comment:
```suggestion
```

Reply:
@Pierre-Sassoulas i do not understand why i should remove this. Can you explain a bit further please?

Reply:
Isn't it a debug print, that would appear when launching pylint on top of the report ?

Reply:
Nope. It is a regular print.